dask/dataframe/core.py

```diff
@@ -209,9 +209,9 @@ def _determine_split_out_shuffle(shuffle, split_out):
             return config.get("dataframe.shuffle.method", None) or "tasks"
         else:
             return False
-    if shuffle is True:
+    if shuffle_method is True:
```

Comment:
That is something we want to deprecate as well, shuffle_method can only be a string the determines the shuffle method

split_out should be used to signal if you want to shuffle or not

Reply:
We can do this as a follow up though

Reply:
Ya, if it's good for you, would like to do this in a follow up.